### ISSUE TITLE
Concurrent modification exception

### ISSUE THREAD

**Issue by OpenSource03** [unldenis/Hologram-Lib#36]
Since the last update, when there are two players online and one removes a hologram, the following error may occur.

```
java.util.ConcurrentModificationException: null
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1605) ~[?:?]
	at java.util.HashMap$KeyIterator.next(HashMap.java:1628) ~[?:?]
	at com.github.unldenis.hologram.HologramPool.remove(HologramPool.java:56) ~[InfiniteForge (2).jar:?]
	at xyz.arcadiadevs.infiniteforge.utils.HologramsUtil.removeHologram(HologramsUtil.java:65) ~[InfiniteForge (2).jar:?]
```

**Comment by unldenis** [comment]
I have a feeling that it refers to #35

**Comment by unldenis** [comment]
It should be fixed by 2.5.0. I tried to simulate this problem with junit tests and it works.
